src/app/coursegrab/utils/__init__.py

```diff
@@ -1 +1,3 @@
 from app.coursegrab.utils.appdev_controller import AppDevController  # noqa: F401
+from app.coursegrab.utils.appdev_redirect_controller import AppDevRedirectController  # noqa: F401
```

Comment:
You could probably use `# flake8: noqa` at the top of file to ignore the entire file